Updating build index: /home/vsts/conda/conda-bld

TEST START: /tmp/tmpfis330xm/noarch/nanocomp-1.15.1-pyhdfd78af_0.tar.bz2
Mamba failed to solve:
 - nanocomp 1.15.1 pyhdfd78af_0
 - nanoplot >=1.35.2
 - joypy
 - matplotlib-base >=2.1.0
 - nanoget >=1.4.0
 - nanomath >=1.0.0
 - pandas
 - plotly >=3.4.2
 - psutil
 - pyarrow
 - python >=3
 - seaborn >=0.9.0
 - python

with channels:
 - file:///tmp/tmpfis330xm

The reported errors are:
   Encountered problems while solving:
     - nothing provides __glibc >=2.17,<3.0.a0 needed by kaleido-core-0.1.0-h3644ca4_0
     - nothing provides __glibc >=2.17,<3.0.a0 needed by kaleido-core-0.1.0-h3644ca4_0
   

Leaving build/test directories:
  Work:
 /home/vsts/conda/conda-bld/work 
  Test:
 /home/vsts/conda/conda-bld/test_tmp 
Leaving build/test environments:
  Test:
source activate  /home/vsts/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh 
  Build:
source activate  /home/vsts/conda/conda-bld/_build_env 


